src/Service/BuildService.php

```diff
@@ -339,6 +339,11 @@ public function addBuildToQueue(Build $build)
     {
         $buildId = $build->getId();
 
+        $this->addJobToQueue($buildId);
+    }
+
+    public function addJobToQueue($buildId, $priority = PheanstalkInterface::DEFAULT_PRIORITY)
```

Comment:
I think we should add new job type like `'php-censor.build` instead of put new type in the `build_id` field. Like:

src/Service/BuildService.php:
```php
public function addJobToQueue($jobType, $buildId = null, $priority = PheanstalkInterface::DEFAULT_PRIORITY)
{
    if ($jobType === BuildWorker::JOB_BUILD && empty($buildId)) {
        return;
    }

    $config   = Config::getInstance();
    $settings = $config->get('php-censor.queue', []);

    if (!empty($settings['host']) && !empty($settings['name'])) {
        try {
            $jobData = [
                'type' => $jobType,
            ];

            if ($jobType === BuildWorker::JOB_BUILD) {
                $jobData['build_id'] = $buildId;
            }

            $pheanstalk = new Pheanstalk(
                $settings['host'],
                $config->get('php-censor.queue.port', Pheanstalk::DEFAULT_PORT)
            );

            $pheanstalk->useTube($settings['name']);
            $pheanstalk->put(
                json_encode($jobData),
                $priority,
                PheanstalkInterface::DEFAULT_DELAY,
                $config->get('php-censor.queue.lifetime', 600)
            );
        } catch (Exception $ex) {
            $this->queueError = true;
        }
    }
}
```

src/Worker/BuildWorker.php:
```php
const JOB_BUILD = 'php-censor.build';
const JOB_STOP  = 'php-censor.stop-flag';

...

protected function verifyJob(Job $job)
{
    $jobData = json_decode($job->getData(), true);

    if (empty($jobData) || !is_array($jobData)) {
        $this->logger->warning(
            sprintf('Empty job (#%s) from the queue tube "%s"!', $job->getId(), $this->queueTube)
        );

        return false;
    }

    $jobType = !empty($jobData['type'])
        ? $jobData['type']
        : '';

    if (self::JOB_STOP === $jobType) {
        $this->stopWorker();

        return false;
    } elseif (self::JOB_BUILD !== $jobType) {
        $this->logger->warning(
            sprintf(
                'Invalid job (#%s) type "%s" in the queue tube "%s"!',
                $job->getId(),
                $jobType,
                $this->queueTube
            )
        );

        return false;
    }

    return true;
}
```

Reply:
I modified to a job type